### PR TITLE
package: update to 2.2.0-1.13.3

### PR DESCRIPTION
**Pull request by dkoshkin**
Updating to upcoming release `2.2.0-1.13.3` and some other minor changes.

- Install latest DC/OS version `1.12.1`
- Add retry looks in Makefile 
- Support using stubs during installation for easier conformance testing
- Using region `us-west-2` which forced us to upgrade to a newer instance type

### CODE REVIEW

File: resources/desired_cluster_profile.aws [hunk 1]
```diff
@@ -3,8 +3,8 @@ num_of_masters = "1"
 num_of_private_agents = "4"
 num_of_public_agents = "1"
 #
-aws_region = "us-west-1"
-aws_bootstrap_instance_type = "m3.large"
+aws_region = "us-west-2"
```
Comment (alejandroEsc) [inline resources/desired_cluster_profile.aws]: +1 awesome

File: Makefile [hunk 2]
```diff
@@ -2,10 +2,12 @@
 RM := rm -f
 SSH_USER := core
 TERRAFORM_INSTALLER_URL := github.com/dcos/terraform-dcos
-DCOS_VERSION := 1.12
-CUSTOM_DCOS_DOWNLOAD_PATH := https://downloads.dcos.io/dcos/stable/1.12.0/dcos_generate_config.sh
-KUBERNETES_VERSION ?= 1.12.2
-KUBERNETES_FRAMEWORK_VERSION ?= 2.0.1-1.12.2
+DCOS_CLI_VERSION := 1.12
```
Comment (alejandroEsc) [inline Makefile]: Why the change in name from DCOS_VERSION to DCOS_CLI_VERSION?

File: Makefile [hunk 3]
```diff
@@ -139,15 +141,26 @@ ui:
...
 		sleep 1; \
 	done
 	@echo "Creating a Kubernetes cluster..."
-	$(DCOS_CMD) kubernetes cluster create --yes --options="$(PATH_TO_PACKAGE_OPTIONS)"
+	$(DCOS_CMD) kubernetes cluster create --yes --options="$(PATH_TO_PACKAGE_OPTIONS)" --package-version="$(KUBERNETES_FRAMEWORK_VERSION)"
+
+.PHONY: add-stubs
+add-stubs:
+ifdef KUBERNETES_STUB_URL
+	@echo "Adding 'kubernetes' stub"
```
Comment (alejandroEsc) [inline Makefile]: Should we not try to remove a previous stub first?

File: Makefile (hunk 2 above)
Comment (dkoshkin) [reply Makefile]: This is actually only used when downloading the CLI, `CUSTOM_DCOS_DOWNLOAD_PATH` is where the DCOS version is defined.

File: Makefile (hunk 3 above)
Comment (dkoshkin) [reply Makefile]: It's being removed as part of the `uninstall`. I don't really feel strongly either way.

Comment (alejandroEsc) [reply Makefile]: ahh i thought you wanted to use this to test stubs.

File: Makefile (hunk 2 above)
Comment (alejandroEsc) [reply Makefile]: Ahh just making it clearer, ok, fair.
